app/api.rb

```diff
@@ -0,0 +1,14 @@
+# Handles web request/response
+require 'sinatra/base'
+require_relative 'food_truck'
+require_relative 'speak_slack'
+
+module FoodTruck
+  class API < Sinatra::Base
+    get '/api' do
```

Comment:
`/api` seems odd for this route.  If/when you add more routes to access more resources, in what sense does it make sense to call this route "api" and not the others?  Should it be called `/available-food-trucks` or `/food-trucks` or something instead?